packages/flet/lib/src/flet_app.dart

```diff
@@ -66,6 +68,7 @@ class _FletAppState extends State<FletApp> {
             extensions: widget.extensions ?? [],
             args: widget.args,
             forcePyodide: widget.forcePyodide,
+            multiView: widget.multiView ?? false,
```

Comment:
```suggestion
            multiView: widget.multiView,
```
multiView can't be null, as defined.